can_testbench.py

```diff
@@ -141,26 +150,35 @@ def sendCanMessage(self, msg, frequency=0):
                     sendDetails['period'] = period
                 else:
                     sendDetails['task'].start()
+            else:
+                sendDetails['task'].start()
```

Comment:
This fixes an existing bug where once you stop sending, you can't send that message again unless you change the frequency.